frappe/core/doctype/version/version.py

```diff
@@ -43,6 +43,8 @@ def get_diff(old, new, for_child=False):
 	if not new:
 		return None
 
+	whitelist_fields = ["Markdown Editor"]
```

Comment:
This is actually the blacklisted fields list according to the code below.

Reply:
You're right.. Fixed it. Thanks!